openapi/gateway.json

```diff
@@ -869,6 +966,15 @@
           "request_id"
         ]
       },
+      "PowerState": {
+        "description": "See RFD 81.\n\nThis enum only lists power states the SP is able to control; higher power states are controlled by ignition.",
+        "type": "string",
+        "enum": [
+          "A0",
+          "A1",
```

Comment:
is this a valid input state or just a valid output state?

Reply:
Depends on what you mean by "valid". It's valid in the sense that MGS will faithfully pass to the `mgmt-gateway` hubris task, which will faithfully pass it to the sequencer task, but invalid in the sense that at least as currently implemented, the sequencer task will return an [illegal transition error](https://github.com/oxidecomputer/hubris/blob/7182c3d6e98a7f3b446fb669feac4de4b4a8e6ac/drv/gimlet-seq-server/src/main.rs#L535) (which will bubble back up to the caller through those same levels).